apps/web/src/app/[locale]/explore/_components/explore-section.tsx

```diff
@@ -65,11 +65,22 @@ export async function ExploreSection({ title, tag, redirectRoute }: SectionProps
       <section className="relative flex w-full flex-col overflow-hidden rounded-[2.5rem]">
         <Carousel>
           {challenges
-            .sort((a, b) =>
-              difficultyToNumber[a.difficulty] !== difficultyToNumber[b.difficulty]
+            .sort((a, b) => {
+              const aHasSubmission = a?.submission?.length && a.submission.length > 0;
```

Comment:
I think we can just do `a?.submission?.length > 0`.

Same thing for `bHasSubmission`

Reply:
you'd have to `(a?.submission?.length ?? 0) > 0` which isnt much better tbh

Reply:
that's fair I guess. I was thinking more along the lines of `undefined > 0` will end up being false anyway.

Reply:
TS wont let you do that

Reply:
Indeed, and imo, I rather have the double check than using the `??` operator here